cmd/auth/token.go

```diff
@@ -21,9 +22,27 @@ func newTokenCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
+
```

Comment:
We seem to have a similar logic in auth command, any chance we can align these two and reuse?

Reply:
Extracted part of it.